src/common/gtest-all.cc

```diff
@@ -6958,7 +6958,7 @@ bool StackLowerThanAddress(const void* ptr) {
 }
 
 bool StackGrowsDown() {
-  int dummy;
+  int dummy = 0;
   return StackLowerThanAddress(&dummy);
```

Comment:
In `StackLowerThanAddress` the same problem occurs: https://github.com/cisco-system-traffic-generator/trex-core/blob/5147003a7fc6cc5859c06d9cf86e6896197d4fb7/src/common/gtest-all.cc#L6956
Maybe this compiler is smart enough to understand that in `StackLowerThanAddress` the value is not in use, but it doesn't understand that for `StackGrowsDown`. Anyway, I suggest you fix that too.

Reply:
Also, since I can't comment on it, why did you make changes on `scripts/so/x86_64/libstdc++.so.6`?

Reply:
I certainly did not mean to :(
Let me re-send!